lib/missions/answers.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:flutter/material.dart';
+
+// Button widget which works as answers to a given Q for missions
+class Answer extends StatelessWidget {
+  // What the answer will do when pressed
+  final Function pressAction;
+
+  final Color textColor;
+
+  Answer(this.pressAction, this.textColor);
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      width: double.infinity,
```

Comment:
Vad gör double.infinity?

Reply:
Tillåter widgeten att sträcka sig så långt som möjligt i width I guess

[https://stackoverflow.com/questions/61706455/whats-the-difference-between-double-infinity-and-double-maxfinite-in-dart](url)